### DISCUSSION TITLE
Welcome to dotfiles Discussions!

### DISCUSSION THREAD

**Discussion by ulises-jeremias** [ulises-jeremias/dotfiles#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kzndotsh** [comment]
This is really great and thorough work, love to see it.

**Comment by ulises-jeremias** [comment]
@kzndotsh thank you very much! ❤️
